decidim-admin/app/commands/decidim/admin/promote_managed_user.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    # A command with all the business logic to promote a managed user.
```

Comment:
Does "promote" mean we are converting the user to a real user? Can you specify this in the docs, please?